kubernetes-archv3/dev-v3-haproxy-service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: dev-v3-haproxy
+spec:
+  selector:
+    cht.service: dev-v3-haproxy
+  clusterIP: None
+  ports:
+  - name: haproxy-port
+    port: 5984
+    targetPort: 5984
+    
```

Comment:
Thanks for adding the newline, but now this one technically still doesn't end with a newline (and nothing else) - this line 13 has spaces followed by the newline. I suspect your editor slipped those in on you. Same goes for a couple other files on the PR. To see which ones need updates, just look here on the GitHub diff and it will show you with this:
![image](https://user-images.githubusercontent.com/1485206/106041541-29a6eb80-6099-11eb-939a-9528a7428b16.png)